### DISCUSSION TITLE
Feature Request - Filament Flow Calibration

### DISCUSSION THREAD

**Discussion by skl111** [5axes/Calibration-Shapes#34] [Ideas]
Hello, Thank you for making this plugin, it is greatly appreciated!
I am requesting a option to Generate this for new filaments for calibration of their flow rate.

https://3dprintbeginner.com/flow-rate-calibration/

**Comment by 5axes** [comment]
You can already generate a Cube.

![image](https://user-images.githubusercontent.com/11015345/111614073-7b3d3e00-87df-11eb-9f5d-b5c068fb70a2.png)
